### ISSUE TITLE
Download

### ISSUE THREAD

**Issue by lhorakov** [OpenSCAP/website#154]
We are going to reorder objects at /Download page. The 2 select labels will be under each other.

EDIT: http://www.open-scap.org/download/

**Comment by jan-cerny** [comment]
Good idea. Consider also slightly adjusting margins and paddings on this page.

**Comment by mpreisler** [comment]
> Consider also slightly adjusting margins and paddings on this page.

It's not clear to me what you mean by this, could you elaborate on how the end result should look like?

**Comment by jan-cerny** [comment]
Yes, I could, I think it should not concentrate everything too near to each other, but use the free space a little better.

**Comment by lhorakov** [comment]
So here is how it looked together.
![screenshot from 2015-12-17 14-31-54](https://cloud.githubusercontent.com/assets/13556662/11870821/5bfbd42c-a4cb-11e5-9b49-cbf52266ed38.png)

It doesn't look good and I realized why in first place I made this order. It was because I wanted to show users first which tool they can select and description about that tool and if they decide to download, then they have there the command line. After testing, none one had problems using Download Page. 

So I am closing it as it is.
